docker-compose.yml

```
@@ -0,0 +1,20 @@
+      - .:/app  # Mount the entire project directory into the container
+    depends_on:
+      - spleeter  # Ensure Spleeter starts before Flask
+
+  spleeter:
+    build:
+      context: ./spleeter_service
+      dockerfile: Dockerfile  # Specifies Dockerfile in the spleeter_service directory
+    ports:
+      - "5001:5001"  # Expose Spleeter on port 5001
+    volumes:
+      - ./spleeter_service/output:/app/output  # Map the output directory for results
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)